### ISSUE TITLE
Help getting app running on Heroku

### ISSUE THREAD

**Issue by patrickbeeson** [portlandpython/portlandpython.org#16]
I'm a little miffed how to proceed here.

I feel like all of the pieces are in place, but something is definitely missing.

Deployment happens automatically on successful builds from github, but for some reason, nothing gets installed python-wise on Heroku. Files are there though.

Any help would be appreciated!

**Comment by hanneshapke** [comment]
Hi Patrick, 

I was testing the Heroku deployment. Two things:
The local deployment test fails and I can't push a non-master branch to heroku. Let's chat about it tomorrow morning.

![image](https://cloud.githubusercontent.com/assets/1234819/12190962/8bdbc7a6-b583-11e5-92e7-886b69c226fb.png)

**Comment by hanneshapke** [comment]
This

```
ImportError: No module named portland_python.config.settings.wsgi
```

seems to be the issue. via `foreman start`

**Comment by patrickbeeson** [comment]
I'm assuming python 3 for this app. I can run it locally without Heroku. But I too am getting an error (different) when running `heroku local`. I can run `heroku local:run python manage.py runserver --settings=config.settings.local` successfully however.

**Comment by patrickbeeson** [comment]
Update: I fixed the bad path to the wsgi file and redeployed. No change in the error at the domain, but I now have pip and things installed on Heroku (confirmed by `heroku run bash`).

I still can't run Heroku local however.

I tried changing the app structure to match their template, but that resulted in the same error. So I don't think that's an issue.

I also tried using virtualenv instead of virtualenvwrapper, but no dice.

I did notice that when I log into the Heroku shell, I can run syncdb using production settings but get an error on the database settings. This looks correct however...

Thoughts?

**Comment by robotbill** [comment]
When Michelle initially created the app we pushed up some other code `git push heroku master` (that didn't work). When I do a `heroku git:clone -a portlandpython` I still get that original code. I wonder if this has anything to do with it.

I'm curious what would happen if we force pushed the new code to `heroku master` for this app.

**Comment by patrickbeeson** [comment]
Thanks @robotbill!

I did a force push to master; everything seemed to work as normal. I then made a small change to settings since I saw an error where `DATABASES` hadn't been declared. After that, I was able to do a migration and create a super user.

But I still can't get gunicorn to run.

The logs say `ImportError: No module named 'config'` but I don't see where this is an issue.

**Comment by alecklandgraf** [comment]
@robotbill @patrickbeeson @hanneshapke 

I'm happy to help out here if #17 doesn't do it.

**Comment by patrickbeeson** [comment]
Go for it! I haven't had the time lately to go back into it.

**Comment by patrickbeeson** [comment]
Closed in #16.
